docs/FAQ.md

```diff
@@ -8,6 +8,16 @@ This document describes common questions about genqlient, and provides an index
 
 There's a [doc for that](INTRODUCTION.md)!
 
+##  … use GET requests instead of POST requests?
+
+You can use `graphql.NewClientUsingGet` to create a client that will use query parameters to create the request. For example:
```

Comment:
It's probably worthwhile to give an example of what this sends over the wire to the server, since it's not obvious what the server would need to do to handle such a request -- unless there's a standard for how to do graphql over GET?  I know there's a standard for POST but I don't know about GET.  (If there is a standard for GET, it may be worth linking to it.)

Reply:
Its here, will link to it

https://graphql.org/learn/serving-over-http/#get-request

Reply:
Yeah, it's semi-standard.  (The official spec has almost nothing about wire format, but the `graphql-js` and Apollo implementations are effectively the standard.)  May as well include the link but in practice I think most servers will either support it or not.

Reply:
Didn't link directly (not sure if this will be stable) but described the format in the FAQ